src/interfaces.ts

```diff
@@ -69,8 +95,7 @@ export interface ISwaggerOptions {
   basePath?: string;
   apiVersion?: '2.0' | '3.0';
   version?: string;
-  auth?: AuthenticationType;
-  authOptions?: AuthenticationOptions;
+  auth?: Authentication;
```

Comment:
This is the breaking change, we now expect this `Authentication` object instead of an `AuthenticationType` to define the root-level `securityDefinitions` and `security` for the Swagger docs.